www/js/commHelper.ts

```diff
@@ -0,0 +1,19 @@
+import { logDebug } from "./plugin/logger";
+
+/**
+ * @param url URL endpoint for the request
+ * @returns Promise of the fetched response (as text) or cached text from local storage
+ */
+export async function fetchUrlCached(url) {
+  const stored = localStorage.getItem(url);
```

Comment:
The nominatim lookup is also cached using localstorage, but in a different way so that we can subscribe to its updates.
(for the key, I used the actual coordinates)

Both implementations are temporary anyway because localstorage is not a good long-term solution and we need to find an alternative eventually (it depends on what we are _allowed_ to use)